curriculum/challenges/english/25-front-end-development/lecture-working-with-arrays/6732aebaa8abb9086a9bb17a.md

```diff
@@ -10,6 +10,43 @@ dashedName: how-do-you-access-and-update-elements-in-an-array
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+How do you access and update elements in an array?
```

Comment:
same here.
We should use double quotes for code examples 

https://contribute.freecodecamp.org/how-to-work-on-coding-challenges/#formatting-seed-code